ext/spl/tests/gh17463.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+GH-15918 (Assertion failure in ext/spl/spl_fixedarray.c)
```

Comment:
Test name is wrong

Reply:
Yes :) will fix the description no worries